pages/services/kafka-zookeeper/2.6.0-3.4.14/security/index.md

```diff
@@ -0,0 +1,299 @@
+   └─ zookeeper-2:[server, metrics] (COMPLETE)
+```
+
+Unauthenticated clients will now only be allowed to ping, create a session, close a session, or authenticate when communicating with the {{ model.techName }} instance.
+
+<p class="message--note"><strong>NOTE: </strong>The default settings for <code>service.security.kerberos.advanced.required_for_quorum_learner</code>, <code>service.security.kerberos.advanced.required_for_quorum_server</code>, <code>service.security.kerberos.advanced.required_for_client</code> are all <code>true</code>.</p>
+
+#### Disabling Kerberos After Deployment
+
+<p class="message--note"><strong>NOTE: </strong> Disabling Kerberos after deployment is <strong>not</strong> supported.
+
+## Securely Exposing DC/OS {{ model.techName }} Outside the Cluster.
+
+Kerberos security is tightly coupled to the DNS hosts of the zookeeper tasks. As such, exposing a secure {{ model.techName }} service outside of the cluster requires additional setup.
+
+### Server to Client Connection
+
+To expose a secure {{ model.techName }} service outside of the cluster, any client connecting to it must be able to access all tasks of the service via the IP address assigned to the task. This IP address will be one of: an IP address on a virtual network or the IP address of the agent the task is running on.
+
+### Forwarding DNS and Custom Domain
+
+Every DC/OS cluster has a unique cryptographic ID which can be used to forward DNS queries to that Cluster. To securely expose the service outside the cluster, external clients must have an upstream resolver configured to forward DNS queries to the DC/OS cluster of the service as described [here](https://docs.mesosphere.com/latest/networking/DNS/mesos-dns/expose-mesos-zone/).
+
+With only forwarding configured, DNS entries within the DC/OS cluster will be resolvable at `<task-domain>.autoip.dcos.<cryptographic-id>.dcos.directory`. However, if you configure a DNS alias, you can use a custom domain. For example, `<task-domain>.cluster-1.acmeco.net`. In either case, the DC/OS {{ model.techName }} service will need to be installed with an additional security option:
+```json
+{
+    "service": {
+        "security": {
+            "custom_domain": "<custom-domain>"
+        }
+    }
+}
+```
+where `<custom-domain>` is one of `autoip.dcos.<cryptographic-id>.dcos.directory` or your organization specific domain (e.g., `cluster-1.acmeco.net`).
+
+As a concrete example, using the custom domain of `cluster-1.acmeco.net` the server 0 task would have a host of `zookeeper-0-server.<service-name>.cluster-1.acmeco.net`.
+
+### Kerberos Principal Changes
+
+With a custom domain endpoint discovery will work as normal. Kerberos, however, does require slightly different configuration. As noted in the section [Create Principals](#create-principals), the principals of the service depend on the hostname of the service. When creating the Kerberos principals, be sure to use the correct domain.
```

Comment:
Write as `Use the correct domain when you create the Kerberos principals.` instead of `When creating the Kerberos principals, be sure to use the correct domain.`.